packages/portfolio-contract/src/portfolio.flows.ts

```diff
@@ -110,7 +260,9 @@ export const openPortfolio = (async (
     };
 
     const openUSDNPosition = async (amount: Amount<'nat'>) => {
-      const acct = kit.keeper.getAccount('USDN');
+      const acct = kit.keeper.getAccount('USDN') as OrchestrationAccount<{
+        chainId: 'noble-any';
+      }>;
```

Comment:
Why add this type annotation? `getAccount('USDN')` already returns that type, no?